owl/robots/PR2.owl

```diff
@@ -287,6 +336,25 @@
     
 
 
+    <!-- http://knowrob.org/kb/PR2.owl#PerceptualAgent -->
+
+    <owl:Class rdf:about="http://knowrob.org/kb/PR2.owl#PerceptualAgent">
```

Comment:
I think this should go somewhere else as it is more genrally usable then just for pr2. maybe into srd2-comp for now?